src/core/api/link_raw.hpp

```diff
@@ -96,7 +96,7 @@ class LinkRaw
      * This method invokes the mTransmitDoneCallback, if set.
      *
      */
-    void InvokeTransmitDone(otRadioFrame *aFrame, bool aFramePending, otError aError);
+    void InvokeTransmitDone(const otRadioFrame *aFrame, const otRadioFrame *aAckFrame, otError aError);
```

Comment:
Thanks, may I finish this documentation task in a coming PR, since I've done part of this work in that PR.